RecoLocalTracker/Phase2TrackerRecHits/python/Phase2StripCPEGeometricESProducer_cfi.py

```diff
@@ -2,7 +2,8 @@
 
 from RecoLocalTracker.Phase2TrackerRecHits.Phase2StripCPEESProducer_cfi import *
 
-phase2StripCPEGeometricESProducer = phase2StripCPEESProducer.clone()
-#phase2StripCPEGeometricESProducer.ComponentName = cms.string('Phase2StripCPEGeometric')
-phase2StripCPEGeometricESProducer.ComponentType = 'Phase2StripCPEGeometric'
-phase2StripCPEGeometricESProducer.parameters    = cms.PSet()
+phase2StripCPEGeometricESProducer = phase2StripCPEESProducer.clone(
+    #ComponentName = 'Phase2StripCPEGeometric',
```

Comment:
Why do we need to keep ComponentName as a comment?

Reply:
In fact, there is not such a `ComponentName` parameter in the `phase2StripCPEESProducer` config: I think it is better if that commmented out line is simply removed